pkg/kubelet/apis/kubeletconfig/types.go

```diff
@@ -39,6 +39,23 @@ const (
 	HairpinNone = "none"
 )
 
+// ResourceChangeDetectionStrategy denotes a mode in which internal
+// managers (secret, configmap) are discovering object changes.
+type ResourceChangeDetectionStrategy string
+
+// Enum settings for different strategies of kubelet managers.
+const (
+	// FetchChangeDetectionStrategy is a mode in which kubelet fetches
+	// necessary objects directly from apiserver.
+	FetchChangeDetectionStrategy ResourceChangeDetectionStrategy = "Fetch"
```

Comment:
Had another thought on this - would it be more standard to call it `Get`, given that this strategy is just sending a GET request to the API server?

Reply:
Makes sense - done.